### PR TITLE
Dropdown issue fixed

### PR DESCRIPTION
**Pull request by smitthakkar96**
Removing dropdown-menu-right class fixes the issue of dropdown being cut.

### CODE REVIEW

**Comment by mbj36** [comment]
When u create any PR, please refer to the issue number mentioned on JIRA or github issue

**Comment by smitthakkar96** [comment]
Sorry I didn't see that the issue number is 1667.

**Comment by mbj36** [comment]
@smitthakkar96  please follow mifos coding style rule, this won't be accepted as you have entered script tag inside html file

File: app/scripts/controllers/client/ClientController.js [hunk 1]
```diff
@@ -55,14 +55,13 @@
                     resourceFactory.globalSearch.search({query: searchString , resource: "clients,clientIdentifiers",exactMatch: exactMatch}, function (data) {
                         var arrayLength = data.length;
                         for (var i = 0; i < arrayLength; i++) {
-                            var result = data[i];
+                            var result = data[i];
```
Comment (devkhan) [inline app/scripts/controllers/client/ClientController.js]: Unnecessary whitespaces. You should consider configuring your editor or git configuration to auto-trim trailing whitespaces.

File: app/scripts/controllers/client/ClientController.js [hunk 2]
```diff
@@ -71,13 +70,13 @@
...
                                 client.displayName = result.parentName;
                                 client.id = result.parentId;
-
                             }
                             scope.actualClients.push(client);
                         }
                         var numberOfClients = scope.actualClients.length;
                         scope.totalClients = numberOfClients;
                         scope.clients = scope.actualClients.slice(0, scope.clientsPerPage);
+                        console.log(scope.clients)
```
Comment (simmimourya1) [inline app/scripts/controllers/client/ClientController.js]: You can remove this.

**Comment by nazeer1100126** [comment]
PR should have a single commit for cleaner history. I don't see any change in ClientController.js. So this file should not be part of the PR.

**Comment by mbj36** [comment]
Conflicts!

**Pull request closed without merging**